bytecomp/bytegen.ml

```diff
@@ -576,7 +576,7 @@ let rec comp_expr env exp sz cont =
         in
         comp_init env sz decl_size
       end
-  | Lprim((Pidentity | Popaque), [arg]) ->
+  | Lprim((Pidentity | Popaque | Pbytes_to_string | Pbytes_of_string), [arg]) ->
```

Comment:
Ok. Then I don't understand why you have some changes in byterun/io.c and byterun/str.c

I understand that you don't want to change the code generation, but in that case you probably don't want to change the runtime either. that would make this revision touch 23 files only